composer.json

```diff
@@ -30,7 +30,7 @@
     "require": {
         "php": ">=8.0",
         "ext-mongodb": "*",
-        "mongodb/mongodb": "1.10.0"
+        "mongodb/mongodb": "^1.21"
```

Comment:
I see there's some 2.x releases, can we test and see if anything breaks?

https://github.com/mongodb/mongo-php-driver/releases